<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>launcher</class>
 <widget class="QDialog" name="launcher">
  <property name="windowModality">
   <enum>Qt::ApplicationModal</enum>
  </property>
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>600</width>
    <height>400</height>
   </rect>
  </property>
  <property name="minimumSize">
   <size>
    <width>600</width>
    <height>400</height>
   </size>
  </property>
  <property name="maximumSize">
   <size>
    <width>600</width>
    <height>400</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>Scoresnake Launcher</string>
  </property>
  <property name="windowIcon">
   <iconset>
    <normalon>:/icons/scoresnake_icon.png</normalon>
   </iconset>
  </property>
  <property name="styleSheet">
   <string notr="true">background-color: #3C4148</string>
  </property>
  <widget class="QLabel" name="visitor_label">
   <property name="geometry">
    <rect>
     <x>40</x>
     <y>115</y>
     <width>200</width>
     <height>20</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>Open Sans</family>
     <pointsize>18</pointsize>
     <weight>75</weight>
     <bold>true</bold>
     <kerning>true</kerning>
    </font>
   </property>
   <property name="styleSheet">
    <string notr="true">color: white;</string>
   </property>
   <property name="text">
    <string>VISITOR</string>
   </property>
   <property name="alignment">
    <set>Qt::AlignCenter</set>
   </property>
   <property name="indent">
    <number>-1</number>
   </property>
  </widget>
  <widget class="QLineEdit" name="visitor_text">
   <property name="geometry">
    <rect>
     <x>40</x>
     <y>140</y>
     <width>200</width>
     <height>40</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>Open Sans</family>
     <pointsize>16</pointsize>
     <weight>75</weight>
     <bold>true</bold>
    </font>
   </property>
   <property name="styleSheet">
    <string notr="true">QLineEdit{
	color: white;
	border: 2px solid #85898f;
}

QLineEdit:focus{
	color: white;
	border: 2px solid white;
}</string>
   </property>
   <property name="alignment">
    <set>Qt::AlignCenter</set>
   </property>
  </widget>
  <widget class="QLabel" name="logo_icon">
   <property name="geometry">
    <rect>
     <x>260</x>
     <y>20</y>
     <width>75</width>
     <height>75</height>
    </rect>
   </property>
   <property name="text">
    <string/>
   </property>
   <property name="pixmap">
    <pixmap resource="Resources/resources.qrc">:/icons/scoresnake_icon_light.png</pixmap>
   </property>
   <property name="scaledContents">
    <bool>true</bool>
   </property>
   <property name="alignment">
    <set>Qt::AlignCenter</set>
   </property>
  </widget>
  <widget class="QLineEdit" name="home_text">
   <property name="geometry">
    <rect>
     <x>360</x>
     <y>140</y>
     <width>200</width>
     <height>40</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>Open Sans</family>
     <pointsize>16</pointsize>
     <weight>75</weight>
     <bold>true</bold>
    </font>
   </property>
   <property name="styleSheet">
    <string notr="true">QLineEdit{
	color: white;
	border: 2px solid #85898f;
}

QLineEdit:focus{
	color: white;
	border: 2px solid white;
}</string>
   </property>
   <property name="alignment">
    <set>Qt::AlignCenter</set>
   </property>
  </widget>
  <widget class="QLabel" name="home_label">
   <property name="geometry">
    <rect>
     <x>360</x>
     <y>115</y>
     <width>200</width>
     <height>20</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>Open Sans</family>
     <pointsize>18</pointsize>
     <weight>75</weight>
     <bold>true</bold>
     <kerning>true</kerning>
    </font>
   </property>
   <property name="styleSheet">
    <string notr="true">color: white;</string>
   </property>
   <property name="text">
    <string>HOME</string>
   </property>
   <property name="alignment">
    <set>Qt::AlignCenter</set>
   </property>
   <property name="indent">
    <number>-1</number>
   </property>
  </widget>
  <widget class="QLabel" name="server_label">
   <property name="geometry">
    <rect>
     <x>200</x>
     <y>205</y>
     <width>200</width>
     <height>20</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>Open Sans</family>
     <pointsize>18</pointsize>
     <weight>75</weight>
     <bold>true</bold>
     <kerning>true</kerning>
    </font>
   </property>
   <property name="styleSheet">
    <string notr="true">color: white;</string>
   </property>
   <property name="text">
    <string>SERVER IP</string>
   </property>
   <property name="alignment">
    <set>Qt::AlignCenter</set>
   </property>
   <property name="indent">
    <number>-1</number>
   </property>
  </widget>
  <widget class="QLineEdit" name="server_ip">
   <property name="geometry">
    <rect>
     <x>200</x>
     <y>230</y>
     <width>200</width>
     <height>40</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>Open Sans</family>
     <pointsize>16</pointsize>
     <weight>75</weight>
     <bold>true</bold>
    </font>
   </property>
   <property name="toolTip">
    <string>IP address of the server running the scorebug.</string>
   </property>
   <property name="styleSheet">
    <string notr="true">QLineEdit{
	color: white;
	border: 2px solid #85898f;
}

QLineEdit:focus{
	color: white;
	border: 2px solid white;
}</string>
   </property>
   <property name="alignment">
    <set>Qt::AlignCenter</set>
   </property>
  </widget>
  <widget class="QPushButton" name="baseball_button">
   <property name="geometry">
    <rect>
     <x>260</x>
     <y>290</y>
     <width>80</width>
     <height>80</height>
    </rect>
   </property>
   <property name="toolTip">
    <string>Launch Baseball Scorekeeper</string>
   </property>
   <property name="styleSheet">
    <string notr="true">QPushButton{
	background-color: rgba(0,0,0,0%);
	border-image: url(:/icons/baseball_outline.png);
	border: 0px;
}

QPushButton:hover{
	background-color: rgba(0,0,0,0%);
	border-image: url(:/icons/baseball_solid.png);
	border: 0px;
}</string>
   </property>
   <property name="text">
    <string/>
   </property>
   <property name="mask" stdset="0">
    <pixmap resource="Resources/resources.qrc">:/icons/baseball_mask.png</pixmap>
   </property>
  </widget>
 </widget>
 <resources>
  <include location="Resources/resources.qrc"/>
 </resources>
 <connections/>
</ui>
